models/public/wav2vec2-base/model.yml

```
@@ -0,0 +1,113 @@
+  - name: wav2vec2-base-960h/vocab.json
+    size: 291
+    sha256: 19727f8944fe6459fc3f240ae2c198395b740f6a029bd23e06656266b83bcf64
+    source: https://huggingface.co/facebook/wav2vec2-base-960h/resolve/main/vocab.json
+  - name: packaging-20.9-py2.py3-none-any.whl
+    size: 40870
+    sha256: 67714da7f7bc052e064859c05c595155bd1ee9f69f76557e21f051443c20947a
+    source: https://files.pythonhosted.org/packages/3e/89/7ea760b4daa42653ece2380531c90f64788d979110a2ab51049d92f408af/packaging-20.9-py2.py3-none-any.whl
+postprocessing:
+  - $type: unpack_archive
+    format: zip
+    file: transformers-4.8.2-py3-none-any.whl
+  - $type: unpack_archive
+    format: zip
+    file: packaging-20.9-py2.py3-none-any.whl
+  - $type: regex_replace
+    file: transformers/__init__.py
+    pattern: 'from . import dependency_versions_check'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: transformers/deepspeed.py
+    pattern: 'from .dependency_versions_check import dep_version_check'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: 'from tqdm.auto import tqdm'
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: from filelock import FileLock
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: from huggingface_hub import HfApi, HfFolder, Repository
+    replacement: '# \g<0>'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: return HfApi\(endpoint=HUGGINGFACE_CO_RESOLVE_ENDPOINT\)\.create_repo\(\n.*\n.*\n.*\n.*\n.*\n.*\n+.*\)
+    replacement: 'return None'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: 'repo = Repository\(repo_path_or_name, clone_from=repo_url, use_auth_token=use_auth_token\)'
+    replacement: 'repo = None'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: 'def _push_to_hub\(cls, repo\: Repository, commit_message\: Optional\[str\] = None\) -> str\:'
+    replacement: 'def _push_to_hub(cls, repo, commit_message: Optional[str] = None) -> str:'
+  - $type: regex_replace
+    file: transformers/file_utils.py
+    pattern: '-> Repository\:'
+    replacement: ':'
+conversion_to_onnx_args:
+  - --model-path=$dl_dir
+  - --model-path=$config_dir
+  - --model-name=create_model
+  - --import-module=model
+  - --model-param=model_dir=r"$dl_dir/wav2vec2-base-960h"
+  - --input-names=inputs
+  - --output-names=logits
+  - --input-shapes=[1,30480]
+  - --output-file=$conv_dir/wav2vec2-base.onnx
+  - '--conversion-param=dynamic_axes={"inputs": {0: "batch_size", 1: "sequence_len"}, "logits": {0: "batch_size", 1: "sequence_len"}}'
+model_optimizer_args:
+  - --input_shape=[1,30480]
+  - --input=inputs
+  - --input_model=$conv_dir/wav2vec2-base.onnx
+  - --output=logits
+framework: pytorch
+quantizable: yes
+license: https://raw.githubusercontent.com/huggingface/transformers/master/LICENSE
```

Maybe need to update license as below
```suggestion
license: https://raw.githubusercontent.com/pytorch/fairseq/master/LICENSE
```